docs/api/sql/Function.md

```diff
@@ -924,6 +924,33 @@ Result:
 +---------------------------------------------------------------+
 ```
 
+
+## ST_Multi
```

Comment:
Please follow the format of other function. 

1. Please add since `1.2.1`.
2. Leave a line break between Example and ```SQL